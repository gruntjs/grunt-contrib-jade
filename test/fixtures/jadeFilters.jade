div
	:some
		test
	:another
		test2