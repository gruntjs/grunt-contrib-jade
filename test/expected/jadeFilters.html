<div><some>test</some><another>test2</another></div>